azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/AADAuthenticationFilter.java

```diff
@@ -81,7 +81,7 @@ protected void doFilterInternal(
                 }
 
                 final Authentication authentication = new PreAuthenticatedAuthenticationToken(
-                            principal, null, client.getGrantedAuthorities(graphApiToken));
+                            principal, null, client.convetGroupToGrantedAuthorities(principal.getUserGroups()));
```

Comment:
@kaykumar Thanks for the PR. Typo `convet`? Change to `convertGroupsToGrantedAuthorities`?

Reply:
@sophiaso I have made this change

Reply:
Thank you!